livingpark_utils/livingpark_utils.py

```diff
@@ -26,6 +26,46 @@
 from PIL import Image
 
 
+def _ensure_column_exists(df: pd.DataFrame, column: str | list[str]) -> None:
+    """Ensure that the `column`(s) exist in the dataframe.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Dataframe to verify.
+    column : str | list[str]
+        Column(s) to verify.
+
+    Raises
+    ------
+    KeyError
+        Column is missing in the dataframe.
+    """
+    if isinstance(column, str):
+        column = [column]
+
+    for c in column:
+        if c not in df.columns:
+            raise KeyError(f'"{c}" column missing in dataframe.')
+
+
+def filter_hc(df: pd.DataFrame) -> pd.DataFrame:
+    """Filter the dataframe to only have Healthy Control.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Dataframe to filter
```

Comment:
Explain that the filtering is done on COHORT and PRIMDIAG and that the dataframe needs these columns

Reply:
I updated the docstring in af08bf8.